novawallet/Modules/Staking/StakingType/StakingTypePresenter.swift

```diff
@@ -324,22 +317,22 @@ extension StakingTypePresenter: StakingSelectPoolDelegate {
 
 extension StakingTypePresenter: StakingSelectValidatorsDelegate {
     func changeValidatorsSelection(validatorList: [SelectedValidatorInfo], maxTargets: Int) {
-        guard let recommendedValidators = recommendedValidators,
-              case let .direct(validators) = method?.selectedStakingOption,
+        guard case let .direct(validators) = method?.selectedStakingOption,
               let restrictions = method?.restrictions else {
             return
         }
         let selectedAddresses = validatorList.map(\.address)
         let selectedValidators = validators.electedValidators.filter {
```

Comment:
This is will not work if we select a validator from the search and it is not in the elected list